src/tests/ftest/pool/create_all_vm.yaml

```diff
@@ -1,10 +1,12 @@
+setup:
+  start_servers_once: False
 hosts:
   test_servers: 5
 timeouts:
   test_one_pool: 240
   test_rank_filter: 240
   test_two_pools: 160
-  test_recycle_pools: 320
+  test_recycle_pools: 1280
```

Comment:
Why was this large timeout bump needed? That seems concerning...

Reply:
Because I have increase by two the number of iteration (from 10 to 20) and we are now testing with two use cases: hugepage enabled or not. Thus, it is why I increase the timeout of 4.

Reply:
Using `!mux` will run each variant (hugepage enabled or not) as a separate test case. Looking here:
https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-11267/5/artifact/Functional%20on%20EL%208/pool/create_all_vm.py/results.html
Each variant took only `430.45s` and `670.13s`

Reply:
Thanks for the tips, I will adjust the timeout according to the time that you gave me.

- [ ] Adjust timeout